src/stores/ScannerContext.ts

```diff
@@ -356,7 +358,7 @@ export function useScannerContext(): ScannerContextState {
 			type: 'transaction'
 		};
 
-		setState(qrInfo);
+		setState({ ...qrInfo, rawPayload: txRequest.data.data });
```

Comment:
We have talked about this sentiment before, but I think in the future comments and updated variable names could be good to avoid confusion. I assumed this is using a useReducer, and it is, but by looking at it right away and having the name be `setState` it could be confusing. I think this is just one overall example of a case where better variable naming could increasingly avoid confusion.